src/Sylius/Bundle/CoreBundle/Controller/ProductController.php

```diff
@@ -271,13 +271,16 @@ public function searchAction(Request $request)
         $results = [];
         $products = $this->container->get('sylius.repository.product')->createFilterPaginator($request->query->get('criteria'));
         $helper = $this->container->get('sylius.templating.helper.currency');
+        $defaultVariantResolver = $this->container->get('sylius.resolver.default_variant');
```

Comment:
In here, yes. But in general, I would suggest to change name of this service from `sylius.resolver.default_variant` to `sylius.variant_resolver.default`